src/run.py

```diff
@@ -60,37 +59,46 @@ def get_sess_dirs(subj_dir_path, mapping_dict):
     return sess_dirs
 
 
+def construct_vqcdeface_cmd(qc_dir):
+    rel_paths_to_orig = [re.sub('/orig.nii.gz', '', str(o.relative_to(qc_dir))) for o in qc_dir.rglob('orig.nii.gz')]
+    with open(qc_dir / 'defacing_id_list.txt', 'w') as f:
+        f.write('\n'.join(rel_paths_to_orig))
+
+    vqcdeface_cmd = f"vqcdeface -u {qc_dir} -i {qc_dir / 'defacing_id_list.txt'} -m orig.nii.gz -d defaced.nii.gz -r defaced_render"
+
+    return vqcdeface_cmd
+
+
 def main():
     # get command line arguments
     args = get_args()
 
-    input_dir = Path(args.bids_dir).resolve()
-    output = Path(args.output_dir).resolve()
+    input_dir = args.bids_dir.resolve()
+    output = args.output_dir.resolve()
     no_clean = args.no_clean
 
     to_deface = []
     # for one subject or list of subjects (and all their sessions, if present)
-    if args.participant_label != None:
+    if args.participant_label is not None:
         for p in args.participant_label:
-            to_deface.extend(glob(os.path.join(args.bids_dir, f'sub-{p}', "ses-*")))
+            to_deface.extend(list(args.bids_dir.joinpath(f'sub-{p}').glob("ses-*")))
+
             if not to_deface:
-                to_deface = glob(os.path.join(args.bids_dir, f'sub-{p}'))
+                to_deface = list(args.bids_dir.glob(f'sub-{p}'))
 
     # only for one subset of sessions
-    if args.session_id != None:
+    if args.session_id is not None:
         for s in args.session_id:
-            to_deface = glob(os.path.join(args.bids_dir, "sub-*", f'ses-{s}'))
+            to_deface = list(args.bids_dir.rglob(f'ses-{s}'))
```

Comment:
rglob 1. These comments are just making sure this was intentional.

Reply:
yes, and same reasoning as above.